oidc-appsupport/src/wasmJsMain/kotlin/org/publicvalue/multiplatform/oidc/appsupport/WasmCodeAuthFlowFactory.kt

```diff
@@ -0,0 +1,9 @@
+package org.publicvalue.multiplatform.oidc.appsupport
+
+import org.publicvalue.multiplatform.oidc.OpenIdConnectClient
+
+class WasmCodeAuthFlowFactory: CodeAuthFlowFactory {
+    override fun createAuthFlow(client: OpenIdConnectClient): PlatformCodeAuthFlow {
+        return PlatformCodeAuthFlow(client)
+    }
```

Comment:
Since this is not used in WASM, maybe it should throw an exception and point to the corrent instructions on how to handle the redirect in wasm?